index.bs

```diff
@@ -2231,6 +2237,139 @@ and makes it clear when the state described by the flags is reset.
+to un-ship it.
+Un-shipping features may result in user and author
+confusion.
+
+<div class=example id=sharedarraybuffer>
+
+When the Spectre attack was discovered,
+{{SharedArrayBuffer}} was unshipped in every implementation
+because it can be used to create a high-precision timer
+that could be used for a Spectre attack.
+
+</div>
+
+Sometimes the same API is implementable in every engine,
+but its behavior cannot be made to be fully interoperable.
+Whenever behavior differs between implementations,
+the end-user impact must be considered.
+Depending on the severity and frequency of the behavior differences,
+this may or may not be an acceptable outcome.
+One downside to this approach
+is that the difference in behavior is not [feature detectable](#feature-detect).
+On the other hand,
+if something changes in the future
+that enables all implementations to converge on the same behavior,
+sites will not need to be updated
+to take advantage of this.
+
+Note: Authors may assume the behavior of the dominant implementation is correct
+and any other behavior is buggy,
+which may further entrench the dominant implementaiton.
+
+<div class=example id=unknown1>
+
+The 'backdrop-filter' property is known to have a number of visible
+behavior differences between implementations.
+At the present time,
+those working on it believe
+the benefits of sharing an API
+outweigh the interoperability costs of the differences.
+
+</div>
+
+If the behavior differences are serious enough,
+it may be preferable to specify different APIs.
+
+In this scenario,
+each implementor goes with the alternative
+that they are able to implement.
+If implementations are eventually able to converge on the same behavior,
+both APIs may need to be supported indefinitely,
+which can increase implementation maintenance costs.
+
+Exposing two different APIs
+that are intended to address the same use cases
+can be confusing to authors.
+Authors can use [feature detection](#feature-detect)
+to write different code for different browsers.
+
+Note: There is a risk that authors may assume
+the API variant supported by the dominant implementation
+is the "correct" one,
+or they may simply be unaware of the API variants supported
+in implementations they do not use regularly,
+which may further entrench the dominant implementaiton.
+
+If you do go this route,
+try to reduce the cost to authors
+by minimizing the differences between the APIs.
+It may be possible to design things
+such that the non-standard part
+is a small, isolated component
+of a larger, standardized framework.
+
+<div class=example id=eme-and-payment-request>
+
+[[ENCRYPTED-MEDIA]] and [[PAYMENT-REQUEST]] are both examples of specifications
+which minimize API differences by isolating a non-standard component
+([=Content Decryption Modules=] and [=payment methods=], respectively)
+from the rest of the (shared) API surface.
+
+</div>
+
+It is also possible to specify an API that not all browsers will implement.
```

Comment:
This paragraph is confusing.

As a start, please at least put the first sentence in the active voice (e.g "you may specify an API...").

But I'm not entirely sure what it's advising, or cautioning against.